command/acl_policy_list.go

```diff
@@ -18,6 +18,10 @@ Usage: nomad acl policy list
 
   List is used to list available ACL policies.
 
+  This command requires a management ACL token to view all policies. A
+  non-management token can be used to list policies associated with that
+  token.
```

Comment:
What's the specific capability required here?  It'd be nice to list it like the rest.

Reply:
That seems weird at first glance, but there isn't one. The token gets access to any policy that's been applied to that token via the [`-policy`](https://nomad-edohxyewc.vercel.app/docs/commands/acl/token-update#policy) flag on the token's create/update. Maybe the wording here needs work?

Reply:
(Or in other words, the token can read its own policy.)

Reply:
Oh I see - my english is failing - can we write something "A non-management token can query its own policies", even though the token doesn't quite do the querying.

Reply:
Reworded that for clarity.